de-de/c-de.html.markdown

```diff
@@ -0,0 +1,880 @@
+---
+language: c
+filename: learnc-de.c
+contributors:
+    - ["caminsha", "https://github.com/caminsha"]
+lang: de-de
+---
+
+Ach, C. Immer noch **die** Sprache für modernes High-Performance Computing.
+
+C ist wahrscheinlich die niedrigste Programmiersprache, welche die meisten
+Programmierer je brauchen werden. Die Geschwindigkeit von C ist enorm, allerdings
+muss man sich stets der maneullen Speicherverwaltung bewusst sein.
+
+
+> **Über Compiler Optionen**
+> 
+> Standardmässig sind `gcc` und `clang` ziemlich ruhig bezüglich Warnungen und 
+> Fehlern, obwohl dies sehr nützliche Informationen sein können. Es wird
+> empfohlen, strengere Compiler Optionen zu verwenden. Hier sind einige empfohlene
+> Standards:
+> `-Wall -Wextra -Werror -O2 -std=c99 -pedantic`
+>
+> Für weitere Informationen, was diese und weitere Optionen genau machen,
+> sollte die Man-Page des C-Compilers aufgerufen werden (z.B. `man 1 gcc`).
+> Alternativ kann auch online nach den unterschiedlichen Optionen gesucht werden.
+
+```c
+// einzeilige Kommentare starten mit // - nur in C99 und später vorhanden.
+
+/*
+mehrzeilige Kommentare sehen so aus. Diese funktionieren auch in C89
+*/
+
+/* 
+mehrzeilige Kommentare können nicht verschachtelt werden /* Sei Vorsichtig! */ // Kommentar endet auf dieser Linie ...
+*/ // ... nicht bei dieser!
+
+// Konstanten: #define <keyword>
+// Konstanten werden laut der Konvention immer in GROSSBUCHSTABEN geschrieben
+#define TAGE_IM_JAHR 365
```

Comment:
even when programming in German, variable and function names themselves should still be in English

Reply:
Changed the variables and functions to English. Makes sense to me.